ts/Core/Scrollbar.ts

```diff
@@ -660,10 +662,12 @@ class Scrollbar {
             options = scroller.options,
             size = scroller.size,
             styledMode = scroller.chart.styledMode,
-            group = renderer.g('scrollbar').attr({
-                zIndex: options.zIndex,
-                translateY: -99999
-            }).add();
+            group = renderer.g('scrollbar')
+                .attr({
+                    zIndex: options.zIndex
+                })
+                .add()
+                .hide(); // initially hide the scrollbar #15863
```

Comment:
If possible, I think it is better to hide it before add. That would prevent rendering and a potential flash of visible content.